sql_generators/mobile_kpi_support_metrics/templates/union.view.sql

```diff
@@ -26,6 +26,11 @@ CREATE OR REPLACE VIEW
     CAST(NULL AS {{ field.type }}) AS {{ field.name }},
   {% endif %}
 {% endfor %}
+{% for field in product.all_possible_attribution_fields if field.exists and field.name == "adjust_network" %}
```

Comment:
@bani I pushed a change which I hope now results in the desired outcome. One thing I wonder, should we be generating this already one step earlier at the individual namespace level rather than the union?

Reply:
Confirmed that it outputs the expected result!